pbnh/tests/paste_curl_test.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/python3
+import pycurl
+import unittest
+import json
+import hashlib
+import tempfile
+import os
+
+from pbnh import conf
+from pbnh import app
+
+config = conf.get_config().get('server')
+host = config.get('bind_ip')
+port = config.get('bind_port')
+URL = 'http://{}:{}/'.format(host, port)
+
+
+class TestPost(unittest.TestCase):
+    def setUp(self):
+        self.c = pycurl.Curl()
+        self.c.setopt(pycurl.URL, URL)
+        self.db_fd, app.app.config['DATABASE'] = tempfile.mkstemp()
```

Comment:
It looks like you aren't actually creating the sqlite database here. It should get created automatically when you first try to access it because it is sqlite, but it would still be good practice to actually create it first.